core/src/main/codegen/templates/Parser.jj

```diff
@@ -1312,20 +1313,26 @@ SqlSelect SqlSelect() :
         where = WhereOpt()
         groupBy = GroupByOpt()
         having = HavingOpt()
+<#if parser.includeQualifyClause>
+        qualify = QualifyOpt()
+<#else>
+        { qualify = null; }
```

Comment:
To simplify the parsing logic a bit, can we assign qualify to null in the constructor and then remove this whole else clause?